requirements.txt

```diff
@@ -4,6 +4,6 @@ coincurve>=8.0.0
 py-solc>=3.0.0
 semver
 mypy-extensions
-web3
+web3>=5.0.0a1
```

Comment:
I'm not sure if we should do this at this point, as this will require the client to update as well.